sky/clouds/aws.py

```diff
@@ -486,9 +486,10 @@ def check_credentials(cls) -> Tuple[bool, Optional[str]]:
             aws_catalog.get_default_instance_type()
         except RuntimeError as e:
             return False, (
-                'Failed to fetch the availability zones for the account. It is '
-                'likely due to permission issues, please check the minimal '
-                'permission required for AWS: https://skypilot.readthedocs.io/en/latest/cloud-setup/cloud-permissions/aws.html'  # pylint: disable=
+                'Failed to fetch the availability zones for the account '
+                f'{identity}. It is likely due to permission issues, please '
```

Comment:
```suggestion
                f'{identity!r}. It is likely due to permission issues, please '
```

Reply:
This should be a list of strings. We probably don't want to add `!r`